hummingbot/strategy/amm_arb/amm_arb_config_map.py

```diff
@@ -121,4 +121,11 @@ def order_amount_prompt() -> str:
         default=False,
         validator=validate_bool,
         type_str="bool"),
+    "use_oracle_conversion_rate": ConfigVar(
```

Comment:
You'll need to set `rate_oracle_pairs` global setting variable, you can see Arbitrage config map for an example.